plugins/utilities.json

```diff
@@ -3,6 +3,101 @@
   "description": "Utilities",
   "plugins_base_url": "https://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugins/utilities",
   "plugins": {
+    "sandbox": {
+      "description": "Spawn, control and bots, change music, add teams, spawn objects, change game values in real time, and much more!",
```

Comment:
"Please add something which tells the user that it works only in local games / HOSTS"

Reply:
"Spawn/control bots, change music, add teams, spawn objects, change game values in real time, and much more! Works only when you are host (local games)"

Something like this?

Reply:
when you click something while you're not the host, it will cuss at you saying you are not the host, except for the music button, you can still change local music in multiplayer games

Reply:
I have discussed this with vishu earlier too

Reply:
Resolve whenever you're ready, or I can modify it np